src/components/vm/overview/cpuModal.tsx

```diff
@@ -249,14 +262,14 @@ export const CPUModal = ({ vm, maxVcpu, models }) => {
                        }>
                 <NumberInput
                     id="machines-vcpu-count-field"
-                    value={count}
-                    onMinus={() => onCountSelect(count - 1)}
-                    onPlus={() => onCountSelect(count + 1)}
-                    onChange={event => onCountSelect(event.target.value)}
+                    value={count === 0 ? "" : count}
+                    onMinus={() => onCountSelect(String(count - 1))}
+                    onPlus={() => onCountSelect(String(count + 1))}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)